spec/game_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'game'
+
+describe Game do
+  let(:game) { double :player_1 => "Jim", :player_weapon => :rock, :computer_weapon => :paper }
```

Comment:
Really like how you've used doubles here its a really clever way of setting up your testing!